AssociationsTool/associationstool.py

```diff
@@ -68,13 +71,16 @@ def __init__(self, dbstate, user, options_class, name, callback=None):
                     (a, b, c, two, value) = ref
                     person2 = dbstate.db.get_person_from_handle(two)
                     name2 = name_displayer.display(person2)
-                    stats_list.append((name1, value, name2))
+                    rel = relationship.get_one_relationship(
+                        dbstate.db, person2, person)
+                    stats_list.append((name1, value, name2, rel))
 
         if uistate:
             titles = [
                 (_('Name'), 0, 200),
                 (_('Type of link'), 1, 200),
                 (_('Of'), 2, 200),
+                (_('Relationship Calculator'), 2, 200),
```

Comment:
Shouldn't this column name be "Relationship to each other" or "Relationship"?

Reply:
Yes, maybe. 

As 'Relationship Calculator' [string already exists on gramps translation domain](https://github.com/gramps-project/gramps/blob/master/po/gramps.pot#L31072) (in memory), localization (localisation) is already done .. OK, a lazy workaround. 
I also wanted to point out that these calculations are generated according to our data via gramps Relationship module.
Textual values set on Associations tab/field are "free". So it could be different to Relation (linked by our data) and could have consistecy issues (filling policy, multiple gedcom imports, etc ...).